Formula/emacs.rb

```diff
@@ -56,6 +56,11 @@ class Emacs < Formula
   end
 
   def install
+    # Mojave uses the Catalina SDK which causes issues like
+    # https://github.com/Homebrew/homebrew-core/issues/46393
+    # https://github.com/Homebrew/homebrew-core/pull/70421
+    ENV["ac_cv_func_aligned_alloc"] = "no" if MacOS.version == :mojave
```

Comment:
You might actually be able to pass this as an argument to `./configure`. It just didn't work for `gcc` because the setting didn't seem to survive the different compilation stages.

Reply:
Ok, I'll let it run with this version to see if it solves the issue, and move it configure in a second run.

Reply:
It worked! 🎉